app/service_utils.py

```diff
@@ -0,0 +1,58 @@
+from flask import current_app
+from .utils import get_json_from_request, \
+    json_has_matching_id, json_has_required_keys, drop_foreign_fields
+from .validation import validate_updater_json_or_400, detect_framework_or_400
+from . import search_api_client, apiclient
+
+
+def validate_and_return_updater_request():
```

Comment:
It's in here as it's request based - so it checks the JSON and then does validate, so validate aspects are in validation and request aspects are in here.

May be need to rename "service_utils" It's not massively focused on services.